src/cljs/ataru/virkailija/application/handlers.cljs

```diff
@@ -554,7 +562,7 @@
     (let [db (assoc-in db [:application :review-autosave] nil)]
       {:db   db
        :http {:method              :get
-              :path                (str "/lomake-editori/api/applications/" application-id)
+              :path                (str "/lomake-editori/api/applications/" application-id "?newest-form=" (boolean (get-in db [:application :newest-form])))
```

Comment:
Ois ehkä siistimpää tyyliin `(str "/lomake-editori/api/applications/" application-id (when (boolean (get-in db [:application :newest-form])) "?newest-form=true" ))`

Reply:
Ja toi `boolean` -funkkari ois tarpeeton em. tyylillä